src/table/column-filter.scss

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  width: fit-content;
+
+  .iui-icon-overlay {
+    position: relative;
+
+    > .iui-input {
+      padding-right: $iui-l * 2;
+    }
+
+    > .iui-borderless {
+      position: absolute;
+      right: 0;
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+
+  .iui-right-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+
+    > .iui-date-time {
```

Comment:
I see it is not applied to all inputs in example.

Reply:
This has been removed in response to [this discussion](https://github.com/iTwin/iTwinUI/pull/51#discussion_r622831663).